Lib/urllib/parse.py

```diff
@@ -364,11 +364,19 @@ def _fix_result_transcoding():
 del _fix_result_transcoding
 
 def urlparse(url, scheme='', allow_fragments=True):
-    """Parse a URL into 6 components:
+    """Parse URL and return a ParseResult.
+    SplitResult is a named 6-tuple composed of:
     <scheme>://<netloc>/<path>;<params>?<query>#<fragment>
-    Return a 6-tuple: (scheme, netloc, path, params, query, fragment).
-    Note that we don't break the components up in smaller bits
-    (e.g. netloc is a single string) and we don't expand % escapes."""
+    The scheme, netloc, path, params, query, fragment
+    of netloc can also be accessed as attributes of the
+    ParseResult object.
```

Comment:
This is implied directly by the previous sentence, since this is the major feature of named tuples. IMO it would be better to omit this extra sentence. (Same goes for `urlsplit()` below.)